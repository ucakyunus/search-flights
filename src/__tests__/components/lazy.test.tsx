import { render } from '@testing-library/react';
import withSuspense from '@/components/lazy';

const MockComponent = () => <div>Mock Component</div>;
const WrappedComponent = withSuspense(MockComponent);

test('withSuspense matches snapshot', () => {
  const { asFragment } = render(<WrappedComponent />);
  expect(asFragment()).toMatchSnapshot();
});
