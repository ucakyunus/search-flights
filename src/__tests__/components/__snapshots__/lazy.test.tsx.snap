// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withSuspense matches snapshot 1`] = `
<DocumentFragment>
  <div>
    Mock Component
  </div>
</DocumentFragment>
`;
